src/components/Footer/Footer.js

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+import workoutImage from './assets/workoutImage.jpg';
+import { Link } from 'react-router-dom';
 
 const Footer = () => {
-  return <div>Footer</div>;
-};
+  return (
+    <div className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-row justify-between items-center font-poppins text-secondary">
```

Comment:
No need for font-poppin here
```suggestion
    <div className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-row justify-between items-center text-secondary">
```